### PR TITLE
fiona/ update  terms and conditions

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

- added new pdfs for;
           - general terms of use
           - trading terms
           - funds & transfers
           - risk disclosure

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — APPROVED** [review]
On project board there is a label and notes for 'don't merge' on the card.

**Comment by fiona-deriv** [comment]
@arome-binary is it possible to delete the redmine link attached to the comment section above please

**Comment by arome-binary** [comment]
> @arome-binary is it possible to delete the redmine link attached to the comment section above please

@fiona-deriv  Link deleted!

**Comment by fiona-deriv** [comment]
thank you

File: src/pages/terms-and-conditions/_clients-grid.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const IconGrid = () => {
             content: localize('What you’re agreeing to when you sign up to use Deriv'),
             non_eu_links: [
                 {
-                    url: '/tnc/general-terms.pdf',
+                    url: '/tnc/general-terms-v1.0.pdf',
```
Comment (ShuvoHabib) [inline src/pages/terms-and-conditions/_clients-grid.js:63]: Hi @fiona-deriv, I think we can just rename the new file. No change in this file is needed.

Comment (fiona-deriv) [reply src/pages/terms-and-conditions/_clients-grid.js:63]: Okay, I'll revert that

**Review by ShuvoHabib — APPROVED** [review]
LGTM ~

**Review by ShuvoHabib — DISMISSED** [review]

**Review by sean-binary — CHANGES_REQUESTED** [review]
As discussed, we need the PDF as shown in production currently. Please do help to make the amendments 🙏

**Review by sean-binary — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**